style.css

```diff
@@ -27,9 +27,28 @@ html {
     background-color: #5471ab;
     height: auto;
     background-image: url("images/image_banner.png");
+    opacity: 50%;
 }
 
-:root {
+.banner-container button {
+    background-color: #5471ab;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 40px;
+    /* background: url(images/image_banner.png); */
+    border-radius: 50;
+    border-radius: 25px;
+    font-weight: bold;
+    position: relative;
+    top: -200px
```

Comment:
should you mean to move this button 200px up from its normal position, please consider that the gap left by it will not be fit by other elments automatically.